docs/source/executing-operations/defer-support.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Apollo Router support for @defer
+description: Incremental delivery of entity fields
+---
+
+> This feature is currently in **preview**, available in Apollo Router v1.0 and later.
+
+Queries sent to the Apollo Router can use the `@defer` directive to enable the incremental delivery of data and improve the router's initial response time. This functionality is contained entirely within the router and does _not_ require your subgraphs to also support `@defer`!
+
+## What is `@defer`?
+
+The `@defer` directive ([see the RFC](https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md)) enables a client query to specify sets of fields that it doesn't need to receive data for _immediately_. This is helpful whenever some fields in a query take much longer to resolve than others.
+
+Here's an example query that uses `@defer`:
+
+```graphql
+query GetPersonAndFriends($personId: ID!) {
+  person(id: $personId) {
+    # Basic fields (fast)
+    id
+    name {
+      firstName
+      lastName
+    }
+
+    # highlight-start
+    # Friend fields (slower)
+    ... on User @defer {
```

Comment:
is there are a particular reason for the conditional fragment? this could just be 
```graphql
... @defer {
  friends {
    id
  }
}
```

Reply:
done!